.github/workflows/python-publish.yml

```diff
@@ -1,6 +1,7 @@
 # This workflows will upload a Python Package using Twine when a release is created
```

Comment:
@lnxpy I think this line has to be updated

Reply:
Fixed!